java/src/jmri/jmrit/beantable/BeanTableBundle.properties

```diff
@@ -818,6 +818,9 @@ LogixNG_CsvFileName                  = CSV file name
 LogixNG_ModuleSocketType             = Socket type:
 LogixNG_typeExternalTable            = External CSV table
 LogixNG_typeInternalTable            = Internal table stored in the panel file
+LogixNG_CsvType_Tabbed               = Tabbed
+LogixNG_CsvType_RFC                  = CSV
```

Comment:
I think this should be Comma.  For most people, CSV is a generic term that supports a variety of export formats, including comma separated and tab separated.  That would be clearer since the row label says CSV type.

Reply:
I agree with @dsand47 .
But these properties should be removed from this file since they are not needed here.